src/NexusMods.App.UI/Controls/Dummy/DummyViewModel.cs

```diff
@@ -13,10 +11,6 @@ public class DummyViewModel : APageViewModel<IDummyViewModel>, IDummyViewModel
 
     public DummyViewModel(IWindowManager windowManager) : base(windowManager)
```

Comment:
Do we still need the dummy view model? I would just remove this (doesn't need to be in this PR)

Reply:
Removed it.